### PR TITLE
Generic sql bug mssql driver

### PR DESCRIPTION
**Pull request by rshunim**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-20180,https://jira-hq.paloaltonetworks.local/browse/CIAC-5068

## Description
This fix should resolve a connection error of 'Microsoft SQL Server  - MS ODBC Driver'.
https://stackoverflow.com/questions/71688125/odbc-driver-18-for-sql-serverssl-provider-error1416f086

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/GenericSQL/Integrations/GenericSQL/GenericSQL.py [hunk 1]
```diff
@@ -284,7 +290,12 @@ def main():
         else:
             raise NotImplementedError(f'{command} is not an existing Generic SQL command')
     except Exception as err:
-        return_error(f'Unexpected error: {str(err)} \nquery: {demisto.args().get("query")} \n{traceback.format_exc()}')
+        if type(err).__name__ == 'OperationalError':
+            return_error("Unexpected error: certificate verify failed, unable to get local issuer certificate. "
+                         "Try selecting 'Trust any certificate' checkbox in the integration configuration.")
+        else:
+            return_error(
+                f'Unexpected error: {str(err)} \nquery: {demisto.args().get("query")} \n{traceback.format_exc()}')
```
Comment (esharf) [inline Packs/GenericSQL/Integrations/GenericSQL/GenericSQL.py:298]:
```suggestion
                f'Unexpected error: {str(err)} \nquery: {demisto.args().get("query")}')
```
the `return_error` method already takes care about it

File: Packs/GenericSQL/ReleaseNotes/1_0_13.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### Generic SQL
+- Fixed an issue with 'Microsoft SQL Server  - MS ODBC Driver' had connection error.
```
Comment (esharf) [inline Packs/GenericSQL/ReleaseNotes/1_0_13.md:4]:
```suggestion
- Fixed an issue with 'Microsoft SQL Server - MS ODBC Driver' had connection error.
```

**Review by esharf — APPROVED** [review]
looks good

File: Packs/GenericSQL/Integrations/GenericSQL/GenericSQL.py [hunk 3]
```diff
@@ -284,7 +289,12 @@ def main():
         else:
             raise NotImplementedError(f'{command} is not an existing Generic SQL command')
     except Exception as err:
-        return_error(f'Unexpected error: {str(err)} \nquery: {demisto.args().get("query")} \n{traceback.format_exc()}')
+        if type(err).__name__ == 'OperationalError':
```
Comment (yuvalbenshalom) [inline Packs/GenericSQL/Integrations/GenericSQL/GenericSQL.py:292]: how do you know this is the only case that you will get error named "OperationalError" seems to me that this is can be a family of different errors.
maybe it is better to check if e substring in the error message matches? like if you have 'certificate' or something like this?(need to check the original error.

in any case, you should print the error content in the end and not to hide it.

**Review by yuvalbenshalom — COMMENTED** [review]
see my comment, and also check the build.

**Review by esharf — APPROVED** [review]

**Review by esharf — APPROVED** [review]
great work

File: Packs/GenericSQL/ReleaseNotes/1_0_13.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### Generic SQL
+- Fixed an issue with 'Microsoft SQL Server - MS ODBC Driver' had connection error.
```
Comment (yuvalbenshalom) [inline Packs/GenericSQL/ReleaseNotes/1_0_13.md:4]:
```suggestion
- Fixed an issue where the integration failed to connect to Microsoft SQL Server.
```

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]

**Pull request merged**